ledger/ledgercore/statedelta.go

```diff
@@ -737,3 +737,8 @@ func (ad *AccountDeltas) GetAllAppResources() []AppResourceRecord {
 func (ad *AccountDeltas) GetAllAssetResources() []AssetResourceRecord {
 	return ad.AssetResources
 }
+
+// GetAllAccounts returns all account NewBalanceRecords
+func (ad *AccountDeltas) GetAllAccounts() []BalanceRecord {
+	return ad.Accts
+}
```

Comment:
I'm not exactly against this or anything, but since Accts is exported, do we need a getter function? Maybe it's so we can mock this somewhere behind an interface?

Reply:
I think it's maybe unnecessary now...originally these fields were all unexported and we had Getters for all of the data except BalanceRecords.